app/src/chat_engine.py

```diff
@@ -21,6 +21,17 @@ class ChatEngineInterface(ABC):
     engine_id: str
     name: str
 
+    # Thresholds that determine which retrieved documents are shown in the UI
+    docs_shown_max_num: int = 5
+    docs_shown_min_score: float = 0.65
```

Comment:
These settings are used by [`format_guru_cards()` in chainlit.py](https://github.com/navapbc/labs-decision-support-tool/pull/37/files#diff-a69583fbe252b197719af65bbc30c0cc6dfc623f161681b5797e3ad28bd9b4f4R122)